### PR TITLE
Smile#63

### PR DESCRIPTION
**Pull request by AnjoSs**
The function POST /tour triggers the Saving of given Json-Data grouped by the receiverZIP-Attribute

### CODE REVIEW

File: sphinx/routes/index.js [hunk 1]
```diff
@@ -10,6 +10,22 @@ router.get('/tour', function (req, res, next) {
   });
 });
 
+router.post('/tour', function (req, res, next){
+    let parcel = req.body.DCParcel;
```
Comment (mlichtblau) [inline sphinx/routes/index.js]: Please use `const` here

File: sphinx/routes/index.js [hunk 2]
```diff
@@ -10,6 +10,22 @@ router.get('/tour', function (req, res, next) {
...
+    console.log(parcel);
+    let dir = ("./public/data/DCParcels/" + parcel.receiverZIP + "/");
```
Comment (mlichtblau) [inline sphinx/routes/index.js]: Please use `const` here

File: sphinx/routes/index.js [hunk 3]
```diff
@@ -10,6 +10,22 @@ router.get('/tour', function (req, res, next) {
...
+    if(!fs.existsSync(dir)) {
+      fs.mkdirSync(dir);
+    }
+    let writeStream = fs.createWriteStream("./public/data/DCParcels/" + parcel.receiverZIP + "/" + parcel.sscc + ".json");
```
Comment (mlichtblau) [inline sphinx/routes/index.js]: Please use `const` here

File: sphinx/routes/index.js (hunk 2 above)
Comment (mlichtblau) [reply sphinx/routes/index.js]: You create the dir variable here, but you don't use it 4 lines below.

File: sphinx/routes/index.js (hunk 3 above)
Comment (mlichtblau) [reply sphinx/routes/index.js]: use `dir` here

File: sphinx/routes/index.js [hunk 4]
```diff
@@ -10,6 +10,22 @@ router.get('/tour', function (req, res, next) {
   });
 });
 
+router.post('/tour', function (req, res, next){
+    let parcel = req.body.DCParcel;
+    console.log(parcel);
```
Comment (mlichtblau) [inline sphinx/routes/index.js]: Remove `console.log` statements before pushing to Github!

File: sphinx/routes/index.js [hunk 5]
```diff
@@ -10,6 +10,22 @@ router.get('/tour', function (req, res, next) {
...
+    let dir = ("./public/data/DCParcels/" + parcel.receiverZIP + "/");
+    if(!fs.existsSync(dir)) {
+      fs.mkdirSync(dir);
+    }
+    let writeStream = fs.createWriteStream("./public/data/DCParcels/" + parcel.receiverZIP + "/" + parcel.sscc + ".json");
+    writeStream.write(JSON.stringify(parcel));
```
Comment (mlichtblau) [inline sphinx/routes/index.js]: I don't know whether it's a good idea to write the files synchronously. But I think for now it's fine

File: sphinx/routes/index.js (hunk 1 above)
Comment (AnjoSs) [reply sphinx/routes/index.js]: Good Point

File: sphinx/routes/index.js (hunk 2 above)
Comment (AnjoSs) [reply sphinx/routes/index.js]: Jup

File: sphinx/routes/index.js (hunk 3 above)
Comment (AnjoSs) [reply sphinx/routes/index.js]: You are obviously right

File: sphinx/routes/index.js (hunk 4 above)
Comment (AnjoSs) [reply sphinx/routes/index.js]: Ups

File: sphinx/routes/index.js (hunk 5 above)
Comment (AnjoSs) [reply sphinx/routes/index.js]: But how U do dis?

**Pull request merged**